### DISCUSSION TITLE
Update macports version to 0.9.14

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#5262] [General]
Hi @oleg-derevenetz , could you please update macports version of the project based on a request from players?

**Comment by oleg-derevenetz** [comment]
Hi @ihhub yes of course, I was going to do this later today. Was distracted by interesting bugs :)

**Reply by ihhub** [reply]
Thank you very much!

**Reply by oleg-derevenetz** [reply]
https://github.com/macports/macports-ports/pull/14563
